pkg/interceptors/cel/cel.go

```diff
@@ -83,10 +84,22 @@ func (w *Interceptor) ExecuteTrigger(request *http.Request) (*http.Response, err
 
 	}
 
+	for _, u := range w.CEL.Overlays {
+		val, err := evaluate(u.Expression, env, evalContext)
+		if err != nil {
+			return nil, fmt.Errorf("failed to evaluate overlay expression '%s': %w", w.CEL.Filter, err)
```

Comment:
Should be `w.CEL.Overlays` or something like that?

Reply:
Good catch, thanks!